src/utils/getTSType.js

```diff
@@ -318,6 +320,40 @@ function handleTSTypeOperator(path: NodePath): ?FlowTypeDescriptor {
   }
 }
 
+function handleTSIndexedAccessType(
+  path: NodePath,
+  typeParams: ?TypeParameters,
+): FlowSimpleType {
+  // eslint-disable-next-line no-undef
+  const objectType: $Shape<FlowObjectSignatureType> = getTSTypeWithResolvedTypes(
+    path.get('objectType'),
+    typeParams,
+  );
+  // eslint-disable-next-line no-undef
+  const indexType: $Shape<FlowLiteralType> = getTSTypeWithResolvedTypes(
+    path.get('indexType'),
+    typeParams,
+  );
+
+  // We only get the signature if the objectType is a type (vs interface)
```

Comment:
Why not for interfaces?

Reply:
@devongovett actually I'm not 100% sure why, but I believe in my trial and error when defining the objectType as an interface, there is no signature property on it. I'm not sure if it is related to the discussion [here](https://github.com/reactjs/react-docgen/issues/366#issuecomment-495715639)?